src/test/mockResultTableStore.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Log } from 'sarif';
+
+export const log = {
+    version: '2.1.0',
+    runs: [{
+        tool: {
+            driver: { name: 'Driver' }
+        },
+        results: [{
```

Comment:
Purpose for having 6 results? If 6 results are really necessary, any way to reduce the repetition?

Reply:
pushing a fix in this PR: https://github.com/microsoft/sarif-vscode-extension/pull/304 👍